test.md

```diff
@@ -185,10 +185,17 @@ TODO: Actually implement the `"spectest"` module, or call out to the supplied on
     rule <instrs> (spectest_trap => .) ~> _M:ModuleDecl ... </instrs>
     rule <instrs> (spectest_trap => .) ~> _A:Assertion  ... </instrs>
 
-    rule <instrs> ( import MOD _ (func OID:OptionalId TUSE:TypeUse) )
-          => #func(... type: TUSE, locals: .LocalDecls, body: spectest_trap .Instrs, metadata: #meta(... id: OID, localIds: .Map))
-          ...
-          </instrs>
+    rule <instrs> ( import MOD _ (func OID:OptionalId (type TIDX)) )
+               => #func(... type: TIDX, locals: [ .ValTypes ], body: spectest_trap .Instrs, metadata: #meta(... id: OID, localIds: .Map))
+               ...
+         </instrs>
+      requires MOD ==K #unparseWasmString("\"spectest\"")
+        orBool MOD ==K #unparseWasmString("\"test\"")
+
+    rule <instrs> ( import MOD _ (func OID:OptionalId (type TIDX) _:TypeDecls) )
+               => #func(... type: TIDX, locals: [ .ValTypes ], body: spectest_trap .Instrs, metadata: #meta(... id: OID, localIds: .Map))
+               ...
+         </instrs>
       requires MOD ==K #unparseWasmString("\"spectest\"")
         orBool MOD ==K #unparseWasmString("\"test\"")
```

Comment:
Can we instead rewrite this to the above case so that we only have one rule? eg.:

```
rule <instrs> ( import MOD _ ( func OID (type TIDX) _:TypeDecls ) ) => (import MOD _ ( func OID ( type TIDX ))) ... </instrs>
  requires ...
```